asserts/snapasserts/snapasserts.go

```diff
@@ -126,3 +126,24 @@ func DeriveSideInfo(snapPath string, db asserts.RODatabase) (*snap.SideInfo, err
 		Developer:   devAcct.Username(),
 	}, nil
 }
+
+// FetchSnapAssertions fetches the assertions matching the snap file digest using the given fetcher.
+func FetchSnapAssertions(f asserts.Fetcher, snapSHA3_384 string) error {
```

Comment:
I don't think it would start from another place, it might start from many though, once we have publisher/developer assertions